content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,46 @@
+| dependencyScanPattern (dsp)     | Scans assets that match the given pattern for missing product dependencies.                                                                                                                                 |                                                                                                 |
+| enableQueryLogging              | Enables logging database queries.                                                                                                                                                                           |                                                                                                 |
+| fileDependencyScanPattern (fsp) | Used with `dependencyScanPattern` to further filter the scan.                                                                                                                                               |                                                                                                 |
+| help (h)                        | Displays all available commands and their description.                                                                                                                                                      |                                                                                                 |
+| sortJobsByDBSourceName          | When enabled, sorts pending jobs with equal priority and dependencies by database source name instead of job ID. Useful for automated tests to process assets in the same order each time.                  |                                                                                                 |
+| truncatefingerprint             | Truncates the fingerprint used for processed assets. Useful if you plan to compress product assets to share on another machine because some compression formats like zip will truncate file mod timestamps. |                                                                                                 |
+| waitOnLaunch                    | Briefly pauses Asset Processor during initialization. Useful if you want to attach a debugger.                                                                                                              | Defaults to off, if True then wait for 20s to aid debugger attach.                              |
+| warningLevel                    | Configure the error and warning reporting level for AssetProcessor. 0 Default, 1 for fatal errors, 2 for fatal errors and warnings.                                                                         | Defaults to 0.                                                                                  |
```

Comment:
Is 0 off? No warnings or fatal errors?

```suggestion
| warningLevel                    | Configure the error and warning reporting level for Asset Processor. 1 for fatal errors, 2 for fatal errors and warnings.                                                                         | 0                                                                                  |
```

Reply:
No 0 is accept the default settings for logging for O3DE, which I believe is log everything at Info level.